MIGRATION.md

```diff
@@ -122,6 +120,81 @@ EventListener(handler=handler_fn_return_dict, event_listener_driver=driver)
 EventListener(handler=handler_fn_return_base_event, event_listener_driver=driver)
 ```
 
+### Removed `BaseEventListener.publish_event` `flush` argument.
+
+`BaseEventListenerDriver.publish_event` no longer takes a `flush` argument. If you need to flush the event, call `BaseEventListenerDriver.flush_events` directly.
+
+#### Before
+
+```python
+event_listener_driver.publish_event(event, flush=True)
+```
+
+#### After
+
+```python
+event_listener_driver.publish_event(event)
+event_listener_driver.flush_events()
+```
+
+### Moved `observable` decorator location.
+
+The `observable` decorator has been moved to `griptape.common.decorators`. Update your imports accordingly.
+
+
+#### Before
+
+```python
+from griptape.common.observable import observable
+```
+
+#### After
+
+```python
+from griptape.common.decorators import observable
+```
+
+### Removed `HuggingFacePipelinePromptDriver.params`
+
+`HuggingFacePipelinePromptDriver.params` has been removed. Use `HuggingFacePipelinePromptDriver.extra_params` instead.
+
+#### Before
+
+```python
+driver = HuggingFacePipelinePromptDriver(
+    params={"max_length": 50}
+)
+```
+
+#### After
+
+```python
+driver = HuggingFacePipelinePromptDriver(
+    extra_params={"max_length": 50}
```

Comment:
there is a lot of other parameter renames that we dont have guides for. is this complex enough to warrant an entry here?

Reply:
Probably not, but I'd rather over-include than under-include on the migration guide. Are there any field renames from this release that I'm missing?

Reply:
all of them i was thinking of were from last release.